panic: runtime error: invalid memory address or nil pointer dereference

Version info:

```
Kapacitor starting, version 0.13.0~rc1, branch master, commit c38245a5d7a356baf426efe254ebff3cd937258f
```

Panic:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x638552]

goroutine 25 [running]:
panic(0xc935c0, 0xc820010060)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/influxdata/kapacitor.(*AlertNode).runAlert(0xc820266b00, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/alert.go:404 +0x20b2
github.com/influxdata/kapacitor.(*AlertNode).(github.com/influxdata/kapacitor.runAlert)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/alert.go:115 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc820266b00, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:128 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:129 +0x53
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,16 @@
 # Changelog
 
+## v0.14.0 [unreleased]
+
+### Release Notes
+
+### Features
+
+
+### Bugfixes
+
+- [#537](https://github.com/influxdata/kapacitor/issue/537): Fix panic in alert node when batch is empty.
+
 ## v0.13.0 [2016-05-11]
 
 ### Release Notes

--- a/alert.go
+++ b/alert.go
@@ -408,6 +408,10 @@ func (a *AlertNode) runAlert([]byte) error {
 	case pipeline.BatchEdge:
 		for b, ok := a.ins[0].NextBatch(); ok; b, ok = a.ins[0].NextBatch() {
 			a.timer.Start()
+			if len(b.Points) == 0 {
+				a.timer.Stop()
+				continue
+			}
 			// Keep track of lowest level for any point
 			lowestLevel := CritAlert
 			// Keep track of highest level and point

